include/RED4ext/Scripting/Natives/entEntityID.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <RED4ext/Common.hpp>
+
+namespace RED4ext::ent
+{
+struct EntityID
+{
+    static constexpr const char* NAME = "entEntityID";
+    static constexpr const char* ALIAS = "EntityID";
+
+    static constexpr uint64_t DynamicUpperBound = 0xFFFFFF;
+    static constexpr uint64_t PersistableLowerBound = 9000000;
+    static constexpr uint64_t PersistableUpperBound = 10000000;
```

Comment:
Is this correct that these are decimal and not hex?

Reply:
Yes, the range is correct.